listen.js

```diff
@@ -1,4 +1,4 @@
-const app = require(`${__dirname}/app`)
+const app = require(`${__dirname}/api/app`)
 
 app.listen(9090, (err) => {
```

Comment:
You might want to get a PORT from process.env (which could defaults to 9090 if that doesn't have a value) when you come to host just on the off chance the hosting doesn't allow for port 9090.